client/js/lounge.js

```diff
@@ -465,7 +471,38 @@ $(function() {
 		return false;
 	});
 
+	sidebar.on("click", ".add-channel", (e) => {
```

Comment:
When I click on "+", that opened the network lobby, marked it as read, and therefore hovered the cursor on <kbd>&times;</kbd> to disconnect the network. Scary stuff lol

#1833 addressed the last point, but I think clicking "+" should not open the lobby. What do you think?

Also, could you try switching the badge and "+" button? I think I will prefer that the button doesn't move based on unread badge. Will see after trying out.

Reply:
> but I think clicking "+" should not open the lobby.

Right. It shouldn't. Probably as simple as an `e.preventDefault()` or however that works now.

Reply:
`return false` will do ;)

Reply:
done